mix.exs

```diff
@@ -10,9 +10,7 @@ defmodule OMG.Umbrella.MixProject do
       deps: deps(),
       preferred_cli_env: [
         coveralls: :test,
-        "coveralls.detail": :test,
-        "coveralls.post": :test,
-        "coveralls.html": :test,
+        "coveralls.circle": :test,
```

Comment:
I use `coveralls.details` and `.html` sometimes locally, please leave those in. 

`.post` is pre-circle era so can go